index.js

```diff
@@ -8,6 +8,8 @@ const { Toolkit } = require("actions-toolkit");
 const GH_USERNAME = core.getInput("GH_USERNAME");
 const COMMIT_MSG = core.getInput("COMMIT_MSG");
 const MAX_LINES = core.getInput("MAX_LINES");
+const TARGET_FILE = core.getInput("TARGET_FILE") || "README.md";
```

Comment:
```suggestion
const TARGET_FILE = core.getInput("TARGET_FILE");
```

Isn't the default value already specified in `action.yml`?

Reply:
In case you have a matrix or similar, I think it's preferable to have the `README.md` as a default value. It is also more secure for wrong or invalid file names (odd characters, ASCII, etc.).